src/cache/cache_las.c

```diff
@@ -1113,8 +1113,8 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
 		 * Remove entries from the lookaside that have aged out and are
```

Comment:
How do we clear the prepared entries once they are committed/rollbacked?
Does aging out takes care of these entries or we need to put extra logic to take care of sweeping prepared updates once not needed.

Reply:
For a transaction to commit / rollback, it will mark the update structures as such, hence the pages modified by the transaction will be in memory. When a transaction commit / rollback is in progress all the pages modified by it will be loaded to memory, so the corresponding las entries will also be added back to page and will be removed from las. Hence no need for extra logic to take care of sweeping prepared updates from las.

Reply:
I was concerned as to at what stage do we remove lookaside entries upon reading them back. I confirmed with @agorrod that here in this case reading prepared updates back for a commit/rollback will cause page to be eventually reconciled in the memory and deleted from lookaside. So, we won't need to sweep prepared updates out of lookaside.